src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/FullRemissionHWFCallbackHandler.java

```diff
@@ -54,12 +56,17 @@ private CallbackResponse fullRemissionHWF(CallbackParams callbackParams) {
             Optional.ofNullable(caseData.getClaimIssuedHwfDetails())
                 .ifPresentOrElse(
                     claimIssuedHwfDetails -> updatedData.claimIssuedHwfDetails(
-                        claimIssuedHwfDetails.toBuilder().remissionAmount(claimFeeAmount).build()
+                        claimIssuedHwfDetails.toBuilder().remissionAmount(claimFeeAmount)
+                            .outstandingFeeInPounds(BigDecimal.ZERO)
+                            .build()
                     ),
                     () -> updatedData.claimIssuedHwfDetails(
-                        HelpWithFeesDetails.builder().remissionAmount(claimFeeAmount).build()
+                        HelpWithFeesDetails.builder().remissionAmount(claimFeeAmount)
+                            .outstandingFeeInPounds(BigDecimal.ZERO)
+                            .build()
                     )
                 );
+            helpWithFeesForTabService.setUpHelpWithFeeTab(updatedData);
```

Comment:
line 69 , line 80 can be replaced by 1 line helpWithFeesForTabService.setUpHelpWithFeeTab(updatedData); outside of if else. plz check

Reply:
Updated from review